inc/sitemaps/class-sitemap-image-parser.php

```diff
@@ -425,6 +393,92 @@ protected function get_absolute_url( $src ) {
 	}
 
 	/**
+	 * Returns the attachments for a gallery.
+	 *
+	 * @param int   $id      The post id.
+	 * @param array $gallery The gallery config.
+	 *
+	 * @return array The selected attachments
+	 */
+	protected function get_gallery_attachments( $id, $gallery ) {
+
+		// When there are posts to include.
+		if ( ! empty( $gallery['include'] ) ) {
+			$_attachments = $this->get_attachments(
+				array(
+					'include'     => $gallery['include'],
+					'numberposts' => 5,
+				)
+			);
+
+			$gallery_attachments = array();
+			foreach ( $_attachments as $key => $val ) {
+				$gallery_attachments[ $val->ID ] = $_attachments[ $key ];
+			}
+
+			return $gallery_attachments;
+		}
+
+		// When $id is empty, just return empty array.
+		if ( empty( $id ) ) {
+			return array();
+		}
+
+		return $this->get_gallery_attachments_for_parent( $id, $gallery );
+	}
+
+	/**
+	 * Returns the attachments for the given id.
+	 *
+	 * @param int   $id      The post id.
+	 * @param array $gallery The gallery config.
+	 *
+	 * @return array The selected attachments.
+	 */
+	protected function get_gallery_attachments_for_parent( $id, $gallery ) {
+		$query = array(
+			'numberposts' => -1,
```

Comment:
`numberposts` is not a valid argument for WP_Query. Use `posts_per_page` instead.